app/views/pages/home.html.haml

```diff
@@ -28,6 +28,14 @@
         %tr
           %th Total # active bookings
           %td=@bookings.where(release_date_time: nil).count
+      %h4 Processing Status
+      %table.table.table-striped.table-responsive.processing-status
+        -@active_bookings.group_by(&:status).each do |status, bookings|
+          %tr{:class => status}
+            %th=status.capitalize
+            %td=bookings.count
+            %td=present_percent(bookings.count, @active_bookings.count)
```

Comment:
This is a nice helper. If this gets merged in before my PR, it might be good to use it in the charge tables as well.